dev-itpro/administration/tenant-admin-center-capacity.md

```diff
@@ -32,15 +32,26 @@ Customers can purchase additional database capacity and environments via their r
 
 Every [!INCLUDE[prod_short](../developer/includes/prod_short.md)] customer with [!INCLUDE[prod_short](../developer/includes/prod_short.md)] Premium or Essential subscriptions can use one production environment and three sandbox environments, at no extra charge.  
 
-> [!NOTE]
-> Existing customers with [!INCLUDE[prod_short](../developer/includes/prod_short.md)] Premium or Essentials subscriptions purchased before October 1st 2020 will keep their existing environment entitlements for a period of 1 year until October 1st 2021 or until their subscription is renewed or ended, whichever occurs last. This won't increase their overall database storage capacity and won't enable extra sandbox environments. These benefits are only activated with the production environments purchased by the customers separately.
-
 Customers can also choose to purchase any number of additional production environments via their CSP partner. Each production environment comes with three additional sandbox environments.  
 
 Production and sandbox environments can be created and used in any country where [!INCLUDE[prod_short](../developer/includes/prod_short.md)] service is available, also in the country where the default [!INCLUDE[prod_short](../developer/includes/prod_short.md)] environments are located. Additional environments can be created by customers, administrators, and partners by using the [!INCLUDE[prodadmincenter](../developer/includes/prodadmincenter.md)]. 
 
 When customer administrators create users in Microsoft 365 Admin Center and assign them [!INCLUDE[prod_short](../developer/includes/prod_short.md)] licenses, each user, by default, gets access to all [!INCLUDE[prod_short](../developer/includes/prod_short.md)] environments (sandbox and production) under the same single [!INCLUDE[prod_short](../developer/includes/prod_short.md)] license, still acting within the scope of their license within each of these environments. Administrators can limit users' access to any particular environment by [changing their permissions](/dynamics365/business-central/ui-define-granular-permissions), or by [removing users' access](/dynamics365/business-central/ui-how-users-permissions#to-remove-a-users-access-to-the-system) within that environment. 
 
+### Customers with Premium or Essentials subscriptions purchased before October 1st 2020
+
+Existing customers with [!INCLUDE[prod_short](../developer/includes/prod_short.md)] Premium or Essentials subscriptions purchased before October 1st 2020 will keep their existing environment entitlements (three production and three sandbox environments) for a period of 1 year until October 1st 2021 or until their subscription is renewed or ended, whichever occurs last. 
+
+Access to two additional production environments during this period won't increase your overall database storage capacity and won't enable extra sandbox environments. These benefits are only activated with the production environments purchased by the customers separately.
```

Comment:
Not sure I understand this. Are you saying that they given access to two additional production environments during this period.